examples/dt/bmo01/data-plane.md

```diff
@@ -0,0 +1,151 @@
+# Deploying the OpenStack dataplane
+
+## Assumptions
+
+- The [control plane](control-plane.md) has been successfully deployed.
+
+## Initialize
+
+Switch to the "openstack" namespace
+
+```bash
+oc project openstack
+```
+
+Change to the dataplane directory
+
+```bash
+cd architecture/examples/dt/bmo/dataplane
+```
+
+### Configure BMO - Provisioning to watch all namespaces
+
+```
+oc patch provisioning provisioning-configuration --type merge -p '{"spec":{"watchAllNamespaces": true }}'
+```
+
+### Configure BMO - Provisioning to use external network for virtual-media
+
+```
+oc patch provisioning provisioning-configuration --type merge -p '{"spec":{"virtualMediaViaExternalNetwork": true }}'
+```
+
+### Create the BareMetalHost CRs
+
+```
+pushd baremetalhosts
+```
+
+Modify the [values.yaml](dataplane/baremetalhosts/values.yaml), for each of the nodes (`leaf0-0`, `leaf0-1`, `leaf1-0` and `leaf1-1`) set:
+- `bmc.address`
+- `bootMACAddress`
+- `rootDeviceHints`
+- `preprovisioningNetworkData`
+
+Modify the [bmc-secret.env](dataplane/baremetalhosts/bmc-secret.env) *env* with the BMC `username` and `password`, for example:
+```
+username=root
+password=S3cr3t
+```
+
+```
+kustomize build > baremetalhosts.yaml
+oc apply -f baremetalhosts.yaml
+```
+
+Wait for BareMetalHosts to reach state: `active`
+
+```
+oc get bmh -w
+
+NAME      STATE        CONSUMER   ONLINE   ERROR   AGE
+leaf0-0   inspecting              false            53s
+leaf0-1   inspecting              false            53s
+leaf1-0   inspecting              false            53s
+leaf1-1   inspecting              false            53s
```

Comment:
I wonder if we should show this, since we are disabling inspection by default

Reply:
Yes, let's drop it. I cleaned it up a little so it is just one line of preparing and available for each node as well.